vscode/microsoft-kiota/src/extension.ts

```diff
@@ -39,7 +40,7 @@ let clientOrPluginKey: string;
 let clientOrPluginObject: ClientOrPluginProperties;
 let workspaceGenerationType: string;
 let config: Partial<GenerateState>;
-
+let deepLinkParams: Record<string, string | undefined> = {};
```

Comment:
Any reason we want to move this outside the activate function scope into a file level variable?

Reply:
It was not accessible from the function I was using

Reply:
Are you sure? It should be available within the registerCommandWithTelemetry(reporter,
      `${treeViewId}.generateClient` function since this function is within the activate function.

Reply:
Must have been a different error that it raised at the time. Pushing a fix